deployments/engine/helm/tiflow/values.yaml

```diff
@@ -15,15 +15,19 @@ master:
   dataStorage: 2Gi
   logStorage: 1Gi
   config: |
-    [framework-metastore-conf]
-    auth.user = "root"
-    auth.passwd = ""
+    [framework-meta]
```

Comment:
/cc @lichunzhu  Please make sure this update is compatible with tiflow operator.

Reply:
okay, we will update this later